docs/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "docs",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "clear": "docusaurus clear",
+    "serve": "docusaurus serve",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "typecheck": "tsc"
+  },
+  "dependencies": {
+    "@docusaurus/core": "2.4.1",
+    "@docusaurus/preset-classic": "2.4.1",
+    "@mdx-js/react": "^1.6.22",
+    "@react-three/fiber": "^8.13.1",
+    "@types/three": "^0.152.1",
+    "clsx": "^1.2.1",
+    "docusaurus-preset-openapi": "^0.6.4",
+    "prism-react-renderer": "^1.3.5",
+    "react": "18",
+    "react-dom": "18",
+    "three": "^0.153.0",
+    "typed.js": "^2.0.12",
+    "url": "^0.11.1"
+  },
+  "devDependencies": {
+    "@docusaurus/module-type-aliases": "2.4.1",
+    "@tsconfig/docusaurus": "^1.0.7",
+    "typescript": "^5.1.3"
+  },
+  "browserslist": {
+    "production": [
+      ">0.5%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
```

Comment:
what is this?

Reply:
https://docusaurus.io/docs/browser-support#default-values